docs/route/meta.md

```diff
@@ -187,7 +187,7 @@ const title = {
 
 This is a list of the current route matches. You have access many things, particularly the meta from the parent matches and data.
 
-It's most useful for merging the parent meta into the child meta since the child-meta value is what will be used:
+It's most useful for merging the parent meta into the child meta since the child meta value is what will be used:
```

Comment:
Realized I was a dummy on the last PR -- the hyphen I added here was wrong, so in this PR I am removing it.

Reply:
👍